roles/common/tasks/common.yml

```diff
@@ -28,12 +24,12 @@
   register: apt_key_change
   changed_when: apt_key_list.stdout != apt_key_change.stdout
 
-- name: Upgrade apt
-  apt:
-    upgrade: yes
-    update_cache: yes
-    cache_valid_time: "{{ apt_cache_valid_time }}"
-  when: "not travis_ci"
+# - name: Upgrade apt
+#   apt:
+#     upgrade: yes
+#     update_cache: yes
+#     cache_valid_time: "{{ apt_cache_valid_time }}"
+#   when: "not travis_ci"
 
 - name: Install packages
   apt:
```

Comment:
I figured a number weren't necessary - I just didn't want to go through them all. But I'll fire up a box and test